apps/frontend/src/modules/surveys/components/studio/create/editor-panel/components/EditorTextInput.tsx

```diff
@@ -11,99 +16,205 @@ export interface EditorTextProps {
     showCharacterCount?: boolean;
     maxCharacterCount?: number;
     onChange: (e: React.ChangeEvent<HTMLInputElement>) => void;
-    options?: Option[];
-    attributes?: Option[];
+    tagOptions?: TagOption[];
     defaultValue?: string;
+    showMention: boolean;
 }
 
 export const EditorTextInput = ({
     label,
     className,
     showCharacterCount = true,
     maxCharacterCount = 500,
-    placeholder,
     defaultValue,
+    tagOptions,
     onChange,
-    ...props
+    placeholder,
+    showMention = false,
 }: EditorTextProps) => {
-    const [atBtnClicked, setAtBtnClicked] = useState(false);
-    const [atIndex, setAtIndex] = useState<number | null>(null);
-    const [selectedOption, setSelectedOption] = useState<Option | null>(null);
-    const [inputValue, setInputValue] = useState(defaultValue || "");
+    const [textContent, setTextContent] = useState(getfromDB(defaultValue ?? "", tagOptions ?? []));
+    const [displayInputField, setDisplayInputField] = useState(false);
+    const [fallbackValue, setFallbackValue] = useState(" ");
+    const [inputFieldPosition, setInputFieldPosition] = useState({ left: 0, bottom: 0 });
```

Comment:
Same here too. It should `const [fallbackFieldPosition, setFallbackFieldPosition] = useState({ left: 0, bottom: 0 });`

Reply:
Done